libs/zephyr-agent/src/lib/auth/login.ts

```diff
@@ -7,7 +7,7 @@ import {
   ze_api_gateway,
 } from 'zephyr-edge-contract';
 import { ZeErrors, ZephyrError } from '../errors';
-import { ZeHttpRequest } from '../http/ze-http-request';
+import { makeRequest } from '../http/http-request';
```

Comment:
Can we have this resolve as `@zephyr-agent/http` or something similar?

Reply:
we could create another package just for this http client, but i dont see a valid reason for it now.

Reply:
There are only 4-6 files where we do http requests, so it's internal. Such resolution is possible, but what for?